README.md

```diff
@@ -133,6 +133,19 @@ alt="Nighthawk" align="left" />
 <br /><br /><br />
 </p>
 
+
+
+<p style="clear:both;">
+<h2><a href="https://layer5.io/projects/catalog">Meshery Catalog</a></h2>
+<a href="">
+  <img src=".github/assets/images/catalog/catalog.svg"
```

Comment:
@goldfishdolphin does this same logo version work in both light and dark modes? I'm not suggesting that it does or doesn't. I'm just confirming.

Reply:
It is a two-tone logo that works fine in dark and light mode. 
This image has been recommended by @leecalcote for the catalog card on the projects page.

Reply:
Ok. 👌